### PR TITLE
Rename `Sign-in Method` -> `Login Flow`

### PR DESCRIPTION
**Pull request by brionmario**
### Purpose

Rename `Sign-in Method` -> `Login Flow` in Application tabs.

### Related Issues
- Address https://github.com/wso2/product-is/issues/19236

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: modules/i18n/src/translations/en-US/portals/console.ts
Comment (pavinduLakshan) [inline modules/i18n/src/translations/en-US/portals/console.ts:1]: Shall we update the translations for other languages as well?

File: apps/console/src/extensions/components/authenticators/email-otp/quick-start.tsx [hunk 1]
```diff
@@ -94,7 +94,7 @@ const EmailOTPQuickStart: FunctionComponent<EmailOTPQuickStartPropsInterface> =
                             i18nKey={ "extensions:develop.identityProviders.emailOTP.quickStart.steps.selectEmailOTP" +
                             ".content" }
                         >
-                            Go to <strong>Sign-in Method</strong> tab and click on <strong>Start with default
+                            Go to <strong>Login Flow</strong> tab and click on <strong>Start with default
```
Comment (pavinduLakshan) [inline apps/console/src/extensions/components/authenticators/email-otp/quick-start.tsx:97]: How about defining a constant for "Login Flow" and using it throughout the app?

Comment (brionmario) [reply apps/console/src/extensions/components/authenticators/email-otp/quick-start.tsx:97]: This is just fallback text, IMO, better not to invest a lot here.

**Review by pavinduLakshan — APPROVED** [review]

File: modules/i18n/src/translations/en-US/portals/console.ts
Comment (brionmario) [reply modules/i18n/src/translations/en-US/portals/console.ts:1]: Fixed with https://github.com/wso2/identity-apps/pull/5384/commits/62420dcd114f42dbd6f2610f87566076e44e9d3d

**Review by JayaShakthi97 — APPROVED** [review]

**Pull request merged**